runtime/src/test_utils.rs

```diff
@@ -111,6 +112,8 @@ pub struct MockRuntime<BS = MemoryBlockstore> {
     pub miner: Address,
     pub base_fee: TokenAmount,
     pub id_addresses: HashMap<Address, Address>,
+    pub delegated_addresses: HashMap<Address, Address>,
+    pub delegated_addresses_source: HashMap<Address, Address>,
```

Comment:
I don't think this second map is needed - these mappings should just go in the id_addresses map instead. These mappings should just replicate what's in the Init actor. I can understand that we need the ID -> delegated address map explicitly because we don't have the actor state tree, which would otherwise encapsulate it.

Reply:
I think you're right, will make this change -- thank you!